README.md

```diff
@@ -873,3 +873,7 @@ This will typically provide instructions for how to support new languages.
 
 In some cases you may need to write a small bit of additional client support, or expose a way for the user to set the server's [configuration options](#configuring-haskell-language-server) and
 for them to configure how the server is started.
+
+### Sponsorship
+
+If you want to contribute financially you could do via open-collective: https://opencollective.com/haskell-language-server
```

Comment:
I would add a few more words to describe how the funds are used, now that we can.
```suggestion
If you want to contribute financially you can do so via [open-collective](https://opencollective.com/haskell-language-server). In the past the funding has been used to sponsor [summer student projects](https://mpickering.github.io/ide/index.html). 
```